include/crypto/rand.h

```diff
@@ -116,6 +116,8 @@ size_t ossl_rand_get_user_entropy(OSSL_LIB_CTX *ctx,
                                   size_t min_len, size_t max_len);
 void ossl_rand_cleanup_entropy(OSSL_LIB_CTX *ctx,
                                unsigned char *buf, size_t len);
+void ossl_rand_cleanup_user_entropy(OSSL_LIB_CTX *ctx,
+                                    unsigned char *buf, size_t len);
 size_t ossl_rand_get_nonce(OSSL_LIB_CTX *ctx,
                            unsigned char **pout, size_t min_len, size_t max_len,
                            const void *salt, size_t salt_len);
```

Comment:
**Side note**: Unfortunately, the function signature of the `EVP_RAND_nonce()` function does not fit well for being used in the `rand_get_nonce` callbacks either. We might need a `EVP_RAND_get_nonce` with the matching signature in the future.
```
int EVP_RAND_nonce(EVP_RAND_CTX *ctx, unsigned char *out, size_t outlen);
```